audit/projects/kubernetes-public/services/bigquery/bigquery.datasets.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "kind": "bigquery#dataset",
+    "id": "kubernetes-public:k8s_prod_readiness",
+    "datasetReference": {
+      "datasetId": "k8s_prod_readiness",
+      "projectId": "kubernetes-public"
+    },
+    "location": "US"
+  },
```

Comment:
https://github.com/kubernetes/k8s.io/pull/2717 partially deployed, still missing the IAM bindings because the group doesn't exist